ecs/run-task/main.go

```diff
@@ -1,15 +1,19 @@
 package main
 
 import (
+	"encoding/json"
+	"os"
+
 	kingpin "github.com/alecthomas/kingpin/v2"
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/service/ecs"
 	"github.com/hamstah/awstools/common"
 )
 
 var (
-	taskDefinition = kingpin.Flag("task-definition", "ECS task definition").Required().String()
-	cluster        = kingpin.Flag("cluster", "ECS cluster").Required().String()
+	taskDefinition   = kingpin.Flag("task-definition", "ECS task definition").Required().String()
+	cluster          = kingpin.Flag("cluster", "ECS cluster").Required().String()
+	taskOverrideJSON = kingpin.Flag("task-override-json", "Path to a JSON file with the task override to use").String()
```

Comment:
sorry I missed it in the previous review, can you rename to overrides (with s) to be consistent with the AWS argument?